CHANGELOG.md

```diff
@@ -79,6 +98,12 @@ All notable changes to the Wazuh app project will be documented in this file.
 
 - Fixed the rendering of tables that contains IPs and agent overview [#5471](https://github.com/wazuh/wazuh-kibana-app/pull/5471)
 - Fixed the agents active coverage stat as NaN in Details panel of Agents section [#5490](https://github.com/wazuh/wazuh-kibana-app/pull/5490)
+  <<<<<<< HEAD
+  =======
```

Comment:
**nitpick**: I think the conflict was not correctly solved.

Reply:
Done